<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="LAPTOP-79OMTUBQ" ignored="0" name="Test" tests="9" failures="2" timestamp="01 May 2022 07:38:27 GMT" time="0.111" errors="0">
  <testcase name="tc1" time="0.021" classname="org.test.First"/>
  <testcase name="tc2" time="0.006" classname="org.test.First">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.002" classname="org.test.First">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.001" classname="org.test.First">
    <skipped/>
  </testcase> <!-- tc2 -->
  <testcase name="tc2" time="0.002" classname="org.test.First">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.First.tc2(First.java:15)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc2 -->
  <testcase name="tc3" time="0.002" classname="org.test.First">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.002" classname="org.test.First">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.003" classname="org.test.First">
    <skipped/>
  </testcase> <!-- tc3 -->
  <testcase name="tc3" time="0.002" classname="org.test.First">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.test.First.tc3(First.java:19)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- tc3 -->
</testsuite> <!-- Test -->
